.codeclimate.yml

```diff
@@ -6,3 +6,8 @@ engines:
         enabled: false
       require-jsdoc:
         enabled: false
+exclude_patterns:
+- "bin/render-pdf.js"
+- "db/migrate/20110919012833_create_pages.rb"
+- "db/migrate/20110501210932_create_feedbacks.rb"
+- "spec/controllers/admin/pages_controller_spec.rb"
```

Comment:
I've also added the pages controller spec to this list. I'm not super happy about it but its been throwing code duplication errors for the edit action tests which are similar but deliberately so.